components/support/nimbus-fml/src/backends/swift/templates/FeatureManifestTemplate.swift

```diff
@@ -36,12 +40,23 @@ public class {{ nimbus_object }} {
     ///
     public let features = {{ nimbus_object }}Features()
 
-    /// Clear all cached values for all features.
-    /// This should be called after `applyPendingExperiments` is finished.
+
+    ///
+    /// Refresh the cache of configuration objects.
+    ///
+    /// For performance reasons, the feature configurations are constructed once then cached.
+    /// This method is to clear that cache for all features configured with Nimbus.
+    ///
+    /// It must be called whenever the Nimbus SDK finishes the `applyPendingExperiments()` method.
+    ///
     public func invalidateCachedValues() {
-        {% for f in self.iter_feature_defs() -%}
+        {%- for f in self.iter_feature_defs() %}
```

Comment:
Could you explain what this hyphen does here? There doesn't seem to be any explanation for it in the Jinja documentation, but I have see this pattern elsewhere, so I'm curious now.

Reply:
Yes, sure; it's for [controlling whitespace][1].

Unlike uniffi, we don't have any formatters as part of our generation step, so it's just tweaking the whitespace.

[1]: https://github.com/djc/askama/blob/f997ba97bd298e219f3286fa792ec10874a34079/book/src/configuration.md